### PR TITLE
Leo kinematics and laser update

### PR DESCRIPTION
**Pull request by leoscholl**
Kinematics changes:
- remove 100s of MB analog cursor sampled at 25khz
- downsample all hand/cursor kinematics to 15hz
- change sampling rate of hand/cursor kinematics to 100hz

Laser changes:
- use digital trigger if present to find laser events

### CODE REVIEW

File: tests/test_data.py
Comment (pavi-rajes) [inline tests/test_data.py:1]: unit tests for test_save_hdf() is failing on my side. All tests related to this pull request passed.

File: aopy/preproc/bmi3d.py [hunk 1]
```diff
@@ -120,9 +121,16 @@ def parse_bmi3d(data_dir, files):
...
         data, metadata = _parse_bmi3d_v1(data_dir, files)
         metadata['bmi3d_parser'] = 1
+    
+    # Keep track of the software version
+    metadata['bmi3d_preproc_date'] = datetime.now()
+    try:
+        metadata['bmi3d_preproc_version'] = version('aopy')
+    except:
+        metadata['bmi3d_preproc_version'] = 'unknown'
+    metadata['bmi3d_source'] = os.path.join(data_dir, files['hdf'])
```
Comment (pavi-rajes) [inline aopy/preproc/bmi3d.py:131]: I like this metadata info on bmi3d

File: tests/test_preproc.py
Comment (pavi-rajes) [inline tests/test_preproc.py:1]: looks like this hdf: \test_proc_eyetracking_short.hdf  is missing and my tests didn't pass. All tests related to this PR has passed though. Please check if all tests are passing on your end once.

**Review by pavi-rajes — APPROVED** [review]

File: tests/test_preproc.py
Comment (leoscholl) [reply tests/test_preproc.py:1]: passing on my end.. but i'd be curious to see the errors it spits out on yours

**Comment by leoscholl** [comment]
I increased the preprocessed sampling rate for cursor, hand, and eye kinematics from 100hz to 1000hz. I think this will give us more flexibility when looking at acceleration, for example. below, at 100hz, the eye acceleration becomes very choppy and it's hard to get good saccade timing. having 1ms resolution will be a lot better i think.
image

**Pull request merged**